### PR TITLE
firmware.volt: responsiveness fix

### PR DESCRIPTION
**Pull request by kulikov-a**
Hi!
finally closes https://github.com/opnsense/core/issues/5635 (i hope ;) ):
make tables responsive (by moving `.table-responsive` to the parent div);
workaround bootstrap "dropdowns inside responsive table" issue by adapting the table height when needed;
don't let the buttons fall apart;

thanks!

### CODE REVIEW

**Comment by fichtner** [comment]
It’s probably fine but from past experiences I’d like to avoid editing the firmware page right before branching stable/23.1 so this will sit here until RC1 is out and land in 23.1.x

**Comment by kulikov-a** [comment]
Thanks! of course it definitely doesn't look urgent )

File: src/opnsense/mvc/app/views/OPNsense/Core/firmware.volt [hunk 1]
```diff
@@ -25,6 +25,17 @@
  # POSSIBILITY OF SUCH DAMAGE.
  #}
 
+<style>
+/* "dropdown in responsive table" issue workaround: display dummy road so dropdowns fits the table */
```
Comment (fichtner) [inline src/opnsense/mvc/app/views/OPNsense/Core/firmware.volt:29]:
```suggestion
    /* "dropdown in responsive table" issue workaround: display dummy road so dropdowns fits the table */
```

File: src/opnsense/mvc/app/views/OPNsense/Core/firmware.volt [hunk 2]
```diff
@@ -25,6 +25,17 @@
...
+    /* "dropdown in responsive table" issue workaround: display dummy road so dropdowns fits the table */
+    @media screen and (max-width: 767px) {
+        .dropdown_helper {
+            display: block !important;
+        }
+    }
+    .dropdown_helper {
+        display: none;
+    }
+</style>
```
Comment (fichtner) [inline src/opnsense/mvc/app/views/OPNsense/Core/firmware.volt:38]:
```suggestion
</style>

```

**Comment by fichtner** [comment]
Merged, thanks!

**Comment by MainServerSpace** [comment]
Appreciate it a lot

**Comment by kulikov-a** [comment]
Thanks!)

**Pull request merged**